src/components/chat/chat-panel.tsx

```diff
@@ -1,29 +1,52 @@
 import { inject, observer } from "mobx-react";
 import React from "react";
 import { BaseComponent } from "../base";
+import { getDisplayTimeDate } from "../../utilities/time";
 import { ChatPanelHeader } from "./chat-panel-header";
+import { CommentCard } from "./comment-card";
 import "./chat-panel.scss";
+import { UserModelType } from "../../models/stores/user";
 
 interface IProps {
   newCommentCount: number;
   onCloseChatPanel:(show:boolean) => void;
 }
 
+interface IState {
+  commentData: { comment: string; timePosted: string; user: UserModelType}[];
+}
+
 @inject("stores")
 @observer
-export class ChatPanel extends BaseComponent<IProps> {
-
+export class ChatPanel extends BaseComponent<IProps, IState> {
   constructor(props: IProps) {
     super(props);
+    this.state = {
+      commentData: [],
+    };
+    this.handlePostComment = this.handlePostComment.bind(this);
```

Comment:
If you define the method with an arrow function below you don't need the `bind` here.